time_tracker_flutter_course/lib/app/sign_in/sign_in_page.dart

```diff
@@ -15,35 +36,45 @@ class SignInPage extends StatelessWidget {
     */
     // Retorna un Future<UserCredencial>
     try {
```

Comment:
@Artwot 

todas estas que se repiten hay que refacturarlas con un Factory

https://dart.academy/creational-design-patterns-for-dart-and-flutter-factory-method/

Reply:
De acuerdo, entonces cierro este PR y me centro en implementar lo de Factory.